### ISSUE TITLE
[Bug] updating to v3.3.0 causes script.skinshorts error on Xonfluence skin

### ISSUE THREAD

**Issue by jmh2002** [drinfernoo/plugin.program.autowidget#97]
[Bug] updating to v3.3.0 causes script.skinshorts error on Xonfluence skin and AutoWidget fails to start / run.

Error occurs with identical installations on both Windows and LibreElec.
Rolling back to v3.2.3 solves the error.

Excerpt from the log after updating. 

```
2021-07-20 10:39:49.683 T:9184  WARNING <general>: CGUIMediaWindow::OnMessage - updating in progress
2021-07-20 10:39:50.096 T:5908    ERROR <general>: EXCEPTION: Unknown addon id 'script.skinshortcuts'.
2021-07-20 10:39:50.098 T:5908    ERROR <general>: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                                    - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                                   Error Type: <class 'RuntimeError'>
                                                   Error Contents: Unknown addon id 'script.skinshortcuts'.
                                                   Traceback (most recent call last):
                                                     File "C:\Users\***\AppData\Roaming\Kodi\addons\plugin.program.autowidget\service.py", line 1, in <module>
                                                       from resources.lib import refresh
                                                     File "C:\Users\***\AppData\Roaming\Kodi\addons\plugin.program.autowidget\resources\lib\refresh.py", line 8, in <module>
                                                       from resources.lib import manage
                                                     File "C:\Users\***\AppData\Roaming\Kodi\addons\plugin.program.autowidget\resources\lib\manage.py", line 11, in <module>
                                                       settings.get_addon_info("profile", addon="script.skinshortcuts")
                                                     File "C:\Users\***\AppData\Roaming\Kodi\addons\plugin.program.autowidget\resources\lib\common\settings.py", line 115, in get_addon_info
                                                       _addon = xbmcaddon.Addon() if not addon else xbmcaddon.Addon(addon)
                                                   RuntimeError: Unknown addon id 'script.skinshortcuts'.
                                                   -->End of Python script error report<--
                                                   
2021-07-20 10:39:50.242 T:5908     INFO <general>: Python interpreter stopped
```

**Comment by drinfernoo** [comment]
Ah, that's interesting... I'll have a look.

**Comment by jmh2002** [comment]
I guess it is because Xonfluence does not use script.skinshortcuts so this script is not installed on my setups.

**Comment by drinfernoo** [comment]
That is why. You *could* install a skin that uses it, so that Skin Shortcuts gets installed, and then uninstall that other skin to save space. That would sidestep the issue for now.

I know what's causing the error, but not a super elegant way of stopping it.

**Comment by jmh2002** [comment]
Ok, for the moment I've rolled back to v3.2.3 to solve the error. 

I believe that the current developer of Xonfluence for Matrix (bsoriano) is currently working on integrating Skin Shortcuts but I'm not yet sure when that version will be released.

**Comment by drinfernoo** [comment]
@jmh2002 For what it's worth, AutoWidget does not *need* Skin Shortcuts.

It checks for the existence of Skin Shortcuts when it is trying to decide which widget metafiles to clear away (due to not being used anymore) during its cleaning cycle.

**Comment by jmh2002** [comment]
Hmm, well, since the update to 3.3.0 this 'checking for the existence of Skin Shortcuts' not only throws the above error, but also stops AutoWidget from running altogether.

**Comment by drinfernoo** [comment]
Correct. I've fixed it in 3.3.1 :+1:

**Comment by jmh2002** [comment]
> Correct. I've fixed it in 3.3.1 👍

Confirmed as fixed with v3.3.2, THANKS :)
